index.d.ts

```diff
@@ -0,0 +1,28 @@
+import { Redis, RedisOptions } from "ioredis";
+
+interface RedisOpts extends RedisOptions {
+    password?: string | undefined | null;
```

Comment:
`?:` means the type can be `undefined`

Reply:
`password` is already defined in `RedisOptions`, it's no need to redefine in here ?